docs_src/utilities/Border.svelte

```diff
@@ -0,0 +1,78 @@
+    <div class="border-radius-s padding-xl width-1of3 margin-l bg--eb">border-radius-s</div>
+    <div class="border-radius--t padding-xl width-1of3 margin-l bg--eb">border-radius--t</div>
+    <div class="border-radius-s--t padding-xl width-1of3 margin-l bg--eb">border-radius-s--t</div>
+    <div class="border-radius--b padding-xl width-1of3 margin-l bg--eb">border-radius--b</div>
+    <div class="border-radius-s--b padding-xl width-1of3 margin-l bg--eb">border-radius-s--b</div>
+  </div>
+</Card>
+
+<Prism language="html">
+  {`<div class="border-radius"></div>
+<div class="border-radius-s"></div>
+<div class="border-radius--t"></div>
+<div class="border-radius-s--t"></div>
+<div class="border-radius--b"></div>
+<div class="border-radius-s--b"></div>`}
+</Prism>
+
+<h3>Border colors</h3>
+<p>Tilgændelige farvevalg der kan tilknyttes border-color</p>
+
+<Card className="bg--graa7">
+  <div class="flex flex-justify--between flex-wrap--wrap" slot="content">
+    <div class="flex flex-align--center border border--black padding-xl width-1of3 margin-l bg--eb"><span class="bg--black margin-s--r" style="border-radius: 100%; width: 20px; height: 20px;"></span>border--black</div>
+    <div class="flex flex-align--center border border--graa1 padding-xl width-1of3 margin-l bg--eb"><span class="bg--graa1 margin-s--r" style="border-radius: 100%; width: 20px; height: 20px;"></span>border--graa1</div>
+    <div class="flex flex-align--center border border--graa2 padding-xl width-1of3 margin-l bg--eb"><span class="bg--graa2 margin-s--r" style="border-radius: 100%; width: 20px; height: 20px;"></span>border--graa2</div>
+    <div class="flex flex-align--center border border--graa3 padding-xl width-1of3 margin-l bg--eb"><span class="bg--graa3 margin-s--r" style="border-radius: 100%; width: 20px; height: 20px;"></span>border--graa3</div>
+    <div class="flex flex-align--center border border--graa4 padding-xl width-1of3 margin-l bg--eb"><span class="bg--graa4 margin-s--r" style="border-radius: 100%; width: 20px; height: 20px;"></span>border--graa4</div>
+    <div class="flex flex-align--center border border--graa5 padding-xl width-1of3 margin-l bg--eb"><span class="bg--graa5 margin-s--r" style="border-radius: 100%; width: 20px; height: 20px;"></span>border--graa5</div>
+    <div class="flex flex-align--center border border--graa6 padding-xl width-1of3 margin-l bg--eb"><span class="bg--graa6 margin-s--r" style="border-radius: 100%; width: 20px; height: 20px;"></span>border--graa6</div>
+    <div class="flex flex-align--center border border--graa7 padding-xl width-1of3 margin-l bg--eb"><span class="bg--graa7 margin-s--r" style="border-radius: 100%; width: 20px; height: 20px;"></span>border--graa7</div>
+    <div class="flex flex-align--center border border--white padding-xl width-1of3 margin-l bg--eb"><span class="bg--white margin-s--r" style="border-radius: 100%; width: 20px; height: 20px;"></span>border--white</div>
```

Comment:
forstår ikke helt span cirklerne her?

Reply:
Hov godt spottet, de bliver heller ikke brugt mere.

Reply:
De er lavet for at tydeliggøre hvilken border farve det er, da det var en smule svært at se på den røde baggrund.

<img width="618" alt="Screenshot 2021-06-21 at 14 43 59" src="https://user-images.githubusercontent.com/7330250/122763910-2b1ebd80-d29f-11eb-9840-54e16143300d.png">

Reply:
så fjern den røde og for den sags skyld grå baggrund og lav border stykker end 1px